app/templates/brief.html

```diff
@@ -49,6 +49,40 @@
     {% endwith %}
   </div>
 </div>
+{% elif brief.status == 'cancelled' %}
+<div class="grid-row">
+  <div class="column-one-whole">
+    {%
+      with
+      type = "temporary-message",
+      heading = "This opportunity was cancelled",
+      message = (
+        "The buyer cancelled this opportunity, for example because they no longer have the budget."
+        + "<br />"|safe
```

Comment:
I recently (yesterday!) learned that you can avoid the need for `|safe` by setting the value beforehand using something like 
```
{% set message %}
The first line<br />
is above the second line.
{% endset %}
``` 

See, e.g. https://github.com/alphagov/digitalmarketplace-supplier-frontend/commit/d8c751f33fc12641623bf725aef9b856514762b1

Reply:
Is it preferable to do it this way?

Reply:
@TheDoubleK I changed that now, do you think the code is ready for merging now?